src/main/resources/application.conf

```diff
@@ -9,6 +9,8 @@ instanceId=${?MESOS_TASK_ID}
 
 ats {
   metricsReporter {
+    reportMetrics = true
+    reportMetrics = ${?REPORT_METRICS}
```

Comment:
don't get it. Didn't we discuss this already? I thought this setting was comming from reference.conf in libats already

Reply:
You are right. I now removed the metrics code which is already in libats.